couchbase/tests/test_couchbase.py

```diff
@@ -61,40 +70,57 @@ def test_query_monitoring_metrics(aggregator, instance_query, couchbase_containe
         aggregator.assert_metric('couchbase.query.{}'.format(mname), tags=CHECK_TAGS, count=1)
 
 
-def assert_basic_couchbase_metrics(aggregator, couchbase_container_ip):
+def assert_basic_couchbase_metrics(aggregator, couchbase_container_ip, extract_device=False):
```

Comment:
Let's just always extract device tags to avoid introducing more hacks

Reply:
but in integration test the device comes inside tags so it would fail unless we extract it too when creating the stub